src/main/resources/templates/lemminx/clientSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "xml": {
```

Comment:
I don't know if the language IDs are correct here as they're not explicitly speicfied in this template's `template.json`. All three languages are case-sensitive, though.